core/network/build.gradle.kts

```
@@ -0,0 +1,16 @@
+@Suppress("DSL_SCOPE_VIOLATION")
+plugins {
+    id("com.teamwable.wable.feature")
+    alias(libs.plugins.google.services)
+}
+android {
+    namespace = "com.teamwable.network"
```

```suggestion
    namespace = "com.teamwable.core.network"
```
core 안 붙이는 걸로 할까요? 다 바꾸기엔 너무 많아서요!

---

핫ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 이걸 빼먹다니.. 나중에 바꾸죠.. 넘 많네요